scripts/run-integration-tests.sh

```diff
@@ -212,6 +213,11 @@ popd
 DEFAULT_INTEGRATION_DURATION=$((SECONDS - START))
 echo "TIMELINE: Default CNI integration tests took $DEFAULT_INTEGRATION_DURATION seconds."
 
+echo "*******************************************************************************"
+echo "Running new integration tests on default CNI version, $ADDONS_CNI_IMAGE"
+echo ""
+sh "$DIR"/test/run-integration-tests.sh
```

Comment:
This is renamed with PR 1812 right? It should be `scripts/test/run-e2e-tests.sh`

Reply:
Hi @jayanthvn, No that script is mainly for running e2e tests while this script runs new integration tests in new-integration folder.